deployment/k8s/workflow-controller_allinone.yaml

```diff
@@ -1,64 +0,0 @@
-apiVersion: v1
```

Comment:
could you update the README.md in order to reflect this change?

Reply:
Will do.